composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "pantheon/terminus-plugin-example",
+    "description": "An example Terminus command",
+    "type": "terminus-plugin",
+    "license": "MIT",
+    "autoload": {
+        "psr-4": { "Pantheon\\TerminusHello": "src" }
+    },
+    "extra": {
+        "terminus": {
+            "compatible-version": "1.*"
```

Comment:
I favor either `^1` or `1.*` since they emphasize the major version more than the range. I don't think we should disallow creative compatible ranges, but I don't want plugin authors to just say `>=1.0` and then assume they're good.

I could be swayed either way on the above two options but I don't actually think `1.*` is terribly ambiguous. The splat is a common wildcard character and I've never seen it used to mean "or greater". I don't think anybody would think that `1.1.*` would also include `1.2`. Maybe I'm wrong.